### DISCUSSION TITLE
Melhorar as warps

### DISCUSSION THREAD

**Discussion by austv** [leogianfagna/IssueTracker#939] [Opinioes]
Criar warps que faltam, como austv-minecraft/AusTv#649, uma warp que mostra as safras, os peixes e também as skins.

Rever cada warp, deixar  mais interativo. Falta warp do guarda roupa (falta um guarda roupa decente na verdade).
Fazer tudo isso para dar uma melhorada legal.

**Comment by leogianfagna** [comment]
![image](https://github.com/user-attachments/assets/df4776a2-3a49-40c9-91d9-e4126076420e)

Incluir para removere esse comandos warps e trocar por /lugares, o que pode levar a uma GUI de lugares do servidor ou de jogadores.
